include/cyclic_reduction.hxx

```diff
@@ -88,43 +91,44 @@ public:
   /// By default not periodic
   void setPeriodic(bool p=true) {periodic=p;}
   
-  /// Set up a single equation to invert
-  void setCoefs(T a[], T b[], T c[]) {
```

Comment:
Should this be deprecated instead of removed? Or is the signature compatible with `Matrix<T>`?

Reply:
Yes I guess this should be deprecated -- I've replaced all uses in the repository but I suppose someone could be using it in their private model. My main issue was how to then construct a `Matrix` from this to pass through to the actual routine.

The same for the solve interfaces I guess.